src/hal/src/window.rs

```diff
@@ -470,7 +470,7 @@ pub trait Swapchain<B: Backend>: fmt::Debug + Any + Send + Sync {
     }
 
     /// Present one acquired image without any semaphore synchronization.
-    unsafe fn present_nosemaphores<'a, C>(
+    unsafe fn present_no_semaphores<'a, C>(
```

Comment:
we should go further and call it `present_without_semaphores`.
Also `submit_nosemaphores` needs to be updated as well

Reply:
Done. r? cc @kvark

Reply:
@Michael-Lfx `quad/main.rs` still uses `submit_nosemaphores`

Reply:
Fixed 10 minutes ago.

Reply:
Sorry it looks like there is one more (at least) `submit_nosemaphores` in `compute/main.rs`

Reply:
https://github.com/gfx-rs/gfx/commit/71fe6b5c0f94b3c530c93532205efb678ee58be5 should work